src/scripts/modules/media/tabs/contents/contents-template.html

```diff
@@ -10,6 +10,51 @@
     </div>
     <div class="result-count" data-l10n-id="textbook-editor-contents-results-matches" data-l10n-args='{"hits":{{resultCount}} }'>{{resultCountText}}</div>
   {{/if}}
-  <div class="toc">
-    {{! TocView}}
-  </div>
+
+  {{#if isBook}}
```

Comment:
@philschatz apparently, on first load this isBook is False, while the one in nav is true. I.e. we see a 'Contents' button, but the TOC is not rendered.